ci-operator/step-registry/cucushift/installer/check/aws/bootstrap-ignition/cucushift-installer-check-aws-bootstrap-ignition-commands.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+REGION="${LEASED_RESOURCE}"
+INFRA_ID=$(jq -r '.infraID' ${SHARED_DIR}/metadata.json)
+
+bucket_name=${INFRA_ID}-bootstrap
+key_name="bootstrap.ign"
+
+echo "Checking if s3://${bucket_name}/${key_name} exists"
+aws --region ${REGION} s3 ls s3://${bucket_name}/${key_name}
```

Comment:
Maybe after the check succeeded, we can clean the S3 bucket in this step?

Reply:
The bucket will be removed by `destroy` command, any concerns about this?

Reply:
Ah, I thought installer would be destroy the preserved bootstrap ignition, if that, pls ignore this comment.

Reply:
This command need aws credential.